src/classwrap.d.ts

```diff
@@ -0,0 +1,14 @@
+export as namespace classwrap
+
+export interface ClassSet {
+  [key: string]: boolean | any | ClassSet
+}
+
+export type Class = string | number | ClassSet
+
+/**
+ * Joins all elements of an array or keys of an object into a string.
+ *
+ * @param classes
+*/
+export default function wrap(classes: Class | Class[]): string
```

Comment:
@Swizz Is it necessary to name the function or is it possible to export the definition anonymously?

If it's possible to export the function declaration anonymously, then that would be preferred in the case of classwrap since we are exporting a default.

Reply:
Yes.

Done this only to follow the implementation : 
```js
export default function wrap(classes, prefix) {
```

Reply:
Well, I called the function `wrap` internally, but it would be nice to export the function anonymously if possible. 

@Swizz